src/containers/BookList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BookList = ({ books }) => (
+  <table className="table">
+    <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Title</th>
+        <th scope="col">Category</th>
+      </tr>
+    </thead>
+    <tbody>
+      {
+        books.map(
+          book => <tr key={book.id}><Book book={book} /></tr>,
+        )
+      }
+    </tbody>
+  </table>
+);
+
+BookList.propTypes = {
+  books: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      title: PropTypes.string.isRequired,
+      category: PropTypes.string.isRequired,
+    }),
+  ).isRequired,
+};
+
```

Comment:
- Our BookList Component should connect to Redux store and fetches books from the state
-  It also uses `mapStateToProps` to fetch books from the `store`
- It displays books list in HTML table using `Book` component